homeassistant/components/pvoutput/sensor.py

```diff
@@ -79,28 +109,46 @@ async def async_setup_entry(
     entry: ConfigEntry,
     async_add_entities: AddEntitiesCallback,
 ) -> None:
-    """Set up a Tailscale binary sensors based on a config entry."""
+    """Set up a PVOutput sensors based on a config entry."""
```

Comment:
```suggestion
    """Set up a PVOutput sensor based on a config entry."""
```

Reply:
No multiple 😉 
It iterates a tuple. The follow up is right after this one :)

Reply:
Then "a" is wrong
```suggestion
    """Set up some PVOutput sensors based on a config entry."""
```